keva/server/src/test/java/com/jinyframework/keva/server/ServerTest.java

```diff
@@ -95,4 +95,21 @@ void del() {
             fail(e);
         }
     }
+
+    @Test
+    void getSetExpire() {
```

Comment:
I think there should be a test cases for updating expire to a lower/higher time as well

Reply:
It will take a longer time to run tests

Reply:
lower/higher as in
expire 100
expire 10
sleep(11)
check

or

expire 10
expire 100
sleep(11)
check
sleep(90)
check

Reply:
Please help me to add those tests while I'm working on other improvements 😄

Reply:
ok I'll add them next PR